combat-trainer.lic

```diff
@@ -197,9 +197,18 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    @dump_junk = settings.dump_junk
+    @dump_delay = 300 # in seconds
```

Comment:
Do we need the delay stored as a variable if it is only used in one place? @rcuhljr @kamarcum thoughts?

The same thing probably applies to `@loot_delay` on line 191.